workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/pages/Project/Routines/Builds/Build.tsx

```diff
@@ -178,14 +178,18 @@ const Build = () => {
 							value: i18n.translate('status'),
 						},
 						{
-							key: 'issues',
-							render: (issues: TestrayCaseResultIssue[]) =>
-								issues.map((caseResultIssue, index) => (
-									<JiraLink
-										issue={caseResultIssue}
-										key={index}
-									/>
-								)),
+							key: 'caseResultsIssues',
+							render: (caseResultsIssues: TestrayIssue[]) => {
+								return caseResultsIssues.map(
+									(caseResultIssue, index) => (
+										<JiraLink
+											issue={caseResultIssue}
+											key={index}
+										/>
+									)
+								);
+							},
+
```

Comment:
```suggestion
							render: (caseResultsIssues: TestrayIssue[]) => 
								caseResultsIssues.map(
									(caseResultIssue, index) => (
										<JiraLink
											issue={caseResultIssue}
											key={index}
										/>
									)
								)

```
Since it's a arrow function and you just have one expression, I believe you don't need the return statement here.

Reply:
Resolved in [b0af134](https://github.com/is-solutions-delivery/liferay-portal/pull/5563/commits/b0af1340c821f9151258a2dafd5787d1321f261f), thanks for the suggestion :fire: